app/create-feed-from-gtfs/service.js

```diff
@@ -1,6 +1,40 @@
 import Ember from 'ember';
 
 export default Ember.Service.extend({
+<<<<<<< HEAD
+  store: Ember.inject.service(),
```

Comment:
Expected ')' and instead saw 'store'.<br>Missing semicolon.<br>Expected '(end)' and instead saw ':'.